vaccine_feed_ingest/runners/wa/state/fetch.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+import requests
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+
+# See https://apim-vaccs-prod.azure-api.net/web/graphql for the full interface w/documention
+QUERY = """
+{
+  searchLocations(
+    searchInput: {
+      # Area of Washinton State
+      radiusMiles: 71362
+      # A very large paging size
+      paging: { pageSize: 1000000, pageNum: 1 }
```

Comment:
I am slightly concerned that they will eventually add a max `pageSize` and this will break. I am going to approve and merge, but if you have the time to go back and implement paging that would make this more robust.

Reply:
I could definitely do that. What do we think is a safe page size though 100? 50? I'm also quite available and could quickly fix this if they implemented one.

Reply:
I think a safe page size would be `1000`, but we can also hold off on implementing paging until it becomes important.